src/common/step-audit.js

```diff
@@ -76,7 +76,7 @@ export class StepAudit extends BaseAudit {
     const payload = destination.formatPayload(stepResult, auditContext, context);
     await sendContinuationMessage({ queueUrl, payload }, context);
 
-    log.info(`Step ${step.name} completed for audit ${audit.getId()} of type ${audit.getAuditType()}, message sent to ${step.destination}`);
+    log.info(`Step ${step.name} completed for audit ${audit.getId()} of type ${audit.getAuditType()}, message sent to ${step.destination}, with message ${JSON.stringify(payload)}`);
```

Comment:
We should try to minimise coralogix logs and only put the ones absolutely necessary